### DISCUSSION TITLE
2021-11-02 Update: Latest Spiralito with Helga

### DISCUSSION THREAD

**Discussion by theo-armour** [konturno/konturno.github.io#7] [Show and tell]
@dashdotdotdashdotdot 

## Full Screen: [Spiralito 2021-11-02]( https://konturno.github.io/sandbox/spiralito/r-2021-11-02/spiralito.html )

* Add "Helga" button

![2021-11-03-helga-2](https://user-images.githubusercontent.com/547626/140017402-7bf01db5-5859-4838-acde-0c87c5b8491e.gif)

There's a ton more I can do here.  I think we need to decide on whether I should build tools that generate static images ( ie rebuilding your R ideas in JavaScript) or whether I should keep running with motion efforts.

**Comment by dashdotdotdashdotdot** [comment]
very good.  We should talk.  I don't always check the gmail account, I more likely to see the me account

**Reply by dashdotdotdashdotdot** [reply]
Theo, I added "buddy" to my repo.  Curious to see what you will do with it. Doug
